store/tikv/mocktikv/cop_handler_dag.go

```diff
@@ -337,6 +373,98 @@ func flagsToStatementContext(flags uint64) *stmtctx.StatementContext {
 	return sc
 }
 
+type mockClientStream struct{}
```

Comment:
1. s/mockClientStream/baseMockStreamClient/
2. add comment about this struct.

Reply:
Go has no concept of inherit, so we don't need `baseMockStreamClient`.